### PR TITLE
docker: add pagarme-node and env configuration

### PR DESCRIPTION
**Pull request by grvcoelho**
## Description

Change base images to pagarme/docker-nodejs:8.9. Fix problem with
node_gyp during docker build add g++, python and make. Reduce size of
image multi-stage build feature from docker.

This closes https://github.com/pagarme/ghostbusters/issues/137

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](../CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation
1. I feel comfortable with the code I wrote, and I'm not ashamed to show it to my friends

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: Dockerfile.server [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
+COPY . /app
 
-EXPOSE 3000
-
-USER quarterback
+WORKDIR /app
 
-CMD ["node", "src/bin/server.js"]
+EXPOSE 3000
+CMD ["node", "src/bin/worker.js"]
```
Comment (claytonsilva) [inline Dockerfile.server]: worker aqui está certo?

Comment (grvcoelho) [reply Dockerfile.server]: arrumado :)

File: Dockerfile.server [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+# Install dependencies and npm dependencies
+RUN apk update && \
+    apk add python make g++ && \
+    npm install --production
 
-WORKDIR $HOME/superbowleto
+FROM pagarme/docker-nodejs:8.9
 
-RUN npm install --production
+ENV APP_NAME 'superbowleto'
```
Comment (otaviopace) [inline Dockerfile.server]: Para que esse APP_NAME?

Comment (otaviopace) [reply Dockerfile.server]: Sabe @claytonsilva ?

Comment (claytonsilva) [reply Dockerfile.server]: sim sim 
usamos ele pra definir qual a tabela que você irá acessar nos secrets:

https://github.com/pagarme/docker-nodejs/blob/master/entrypoint.sh#L6

junto com APP_ENV montamos a tabela que contém os dados sensíveis

**Pull request merged**